python/ray/dashboard/modules/actor/actor_head.py

```diff
@@ -2,19 +2,15 @@
 import logging
 import os
 import time
-
 from collections import deque
 
 import aiohttp.web
 
 import ray.dashboard.optional_utils as dashboard_optional_utils
 import ray.dashboard.utils as dashboard_utils
 from ray._private.gcs_pubsub import GcsAioActorSubscriber
-from ray.core.generated import (
-    gcs_service_pb2,
-    gcs_service_pb2_grpc,
-)
-from ray.dashboard.datacenter import DataSource, DataOrganizer
+from ray.core.generated import gcs_service_pb2, gcs_service_pb2_grpc
```

Comment:
are these import changes auto-fixed by a linter or tool?

Reply:
yes.. let me try to split these into a separate PR.

`dashboard/` dir was previously not included in the formatter..

Reply:
https://github.com/ray-project/ray/pull/46483 PR applying isort without moving the files.

we can approve that first.

Reply:
updated, now `dashboard/` files are pure location moving.